src/interface/src/styleguide/filter-dropdown/filter-dropdown.component.html

```diff
@@ -11,22 +11,24 @@
     <div
       class="dropdown-label"
       [ngClass]="{ 'active-filter': hasSelections(), disabled: disabled }">
-      {{ selectionText }}
+      <span>{{ menuLabel }}</span
+      ><span class="selection-blurb">{{ selectionText }}</span>
     </div>
   </div>
-  <div *ngIf="showCount()" class="more-items-box">
-    <div class="more-items-icon">&plus;</div>
```

Comment:
When the button is small, there's not much space to show the text label from a first selected element and a + sign. 
I'm not sure if it's better to avoid displaying it completely, or just when the button is small.

Reply:
not sure i get it, do you mind posting some screenshots?